chef/cookbooks/provisioner/templates/suse/crowbar_register.erb

```diff
@@ -152,19 +152,21 @@ if test "x$MD5_ADMIN" != "x$MD5_LOCAL"; then
     exit 1
 fi
 
+#we need to know an architecture we are running on
+ARCH=`uname -m`
```

Comment:
You need to add an empty line after this one, to keep the two empty lines before the next code section.